services/provider/server/server.go

```diff
@@ -43,11 +43,11 @@ import (
 )
 
 const (
-	TicketAnnotation             = "ocs.openshift.io/provider-onboarding-ticket"
-	ProviderCertsMountPoint      = "/mnt/cert"
-	onboardingTicketKeySecret    = "onboarding-ticket-key"
-	storageClassRequestNameLabel = "ocs.openshift.io/storageclassrequest-name"
-	notAvailable                 = "N/A"
+	TicketAnnotation           = "ocs.openshift.io/provider-onboarding-ticket"
+	ProviderCertsMountPoint    = "/mnt/cert"
+	onboardingTicketKeySecret  = "onboarding-ticket-key"
+	storageClassClaimNameLabel = "ocs.openshift.io/storageclassclaim-name"
```

Comment:
```suggestion
storageClaimNameLabel = "ocs.openshift.io/storageclaim-name"
```
it should be storageClaimNameLabel?

Reply:
I think this is more dangerous (and might have side effects) and regardless should not be part of this PR